Conduit/Conduit/Program.cs

```diff
@@ -22,10 +22,10 @@
     // options.UseQueryTrackingBehavior(QueryTrackingBehavior.NoTracking);
 });
 builder.Services.AddScoped<IAuthService, AuthService>();
-
+builder.Services.AddScoped<IArticlesRepositry, ArticleRepositry>();
```

Comment:
NIT, no need to update anything here, you can register the services by using the attribute [RegisterAs(.....] it looks more clean, especially in case you have a large number of services